Missions.rte/Activities/Doainar.lua

```diff
@@ -279,12 +279,12 @@ function DoainarMission:UpdateActivity()
 	for actor in MovableMan.Actors do
 		if self.aggress == false then
 			if not self.caveAreaA:IsInside(actor.Pos) and not self.caveAreaB:IsInside(actor.Pos) and actor.PresetName == "Crab" then
-				self:AddObjectivePoint("Kill!", actor.AboveHUDPos, Activity.TEAM_1, GameActivity.ARROWDOWN);
+				self:AddObjectivePoint("Kill!", actor.Pos + Vector(0, -actor.Radius), Activity.TEAM_1, GameActivity.ARROWDOWN);
```

Comment:
Hmmm, I wonder if AboveHUDPos should account for radius? Maybe not, since it is supposed to give you the point right above the actor's HUD.
Also, edge-case problem with doing this is if you have a very wide, short actor, your arrow's gonna be pretty high up. But that's pretty uncommon so it's not really not a big deal.

Reply:
This is literally just crabs, and they have no HUD, so the marker overlaps all ugly over their bodies. Maybe non-HUD actors could rely on radius? (or have additional hudstack?)

Reply:
Ahh good point. I think it'd be reasonable to do the radius thing in cpp for now. Wanna scope creep your source PR a tiny bit? :D